discord/guild.py

```diff
@@ -606,6 +613,13 @@ def edit(self, **fields):
             raise InvalidArgument('verification_level field must of type VerificationLevel')
 
         fields['verification_level'] = level.value
+
+        if 'default_message_notifications' in fields:
+            if not isinstance(fields['default_message_notifications'], DefaultMessageNotifications):
```

Comment:
Instead of doing `'key' in fields` and then doing `fields['key']` I implore you to use `try...except` instead as it is much faster.